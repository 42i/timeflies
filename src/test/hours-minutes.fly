wp a
wp b
wp c
wp d
day 2012-07-17 8:30 17:30
sick 3:45
day 2012-07-18 7:45 17:30, off 0:45
- a 2:15; two and a quarter
day 2012-07-19 7:45 17:30, off 0:45
leave 1:30
- b 1; full hour
- c 2.25; two decimals
- d 0.5; one decimal