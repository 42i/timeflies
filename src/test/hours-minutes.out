Time at work overview (all):
     when        worked   leave    sick balance
2012-07-17 Tue:    9.00 ----.--    3.75    4.75
2012-07-18 Wed:    9.00 ----.-- ----.--    1.00
2012-07-19 Thu:    9.00    1.50 ----.--    2.50
  week 2012-29:   27.00    1.50    3.75    8.25
 month 2012-07:   27.00    1.50    3.75    8.25
         total:   27.00    1.50    3.75    8.25
     when        worked   leave    sick balance
Work package summary (all):
   6.00 : ALL
       2.25 : a
       1.00 : b
       2.25 : c
       0.50 : d
